install/linux/docker-ee/ubuntu.md

```diff
@@ -136,11 +136,16 @@ from the repository.
       ```
 
 4. Temporarily add a `$DOCKER_EE_VERSION` variable into your environment.
-    There are currently two versions of Docker EE Engine available:
 
-    * `stable-18.03` - Use this version if you're only running Docker EE Engine.
-    * `stable-17.06` - Use this version if you're using Docker Enterprise Edition (Docker
-    Engine, UCP, and DTR).
+
+    The current version of Docker EE Engine is 18.09 for using with Docker Enterprise Edition 2.1.
+
+    > ***NOTE:*** There are two older versions of Docker EE Engine available:
+    > * 18.03 - Use this version if you're only running Docker EE Engine.
```

Comment:
@ahh-docker Do we need to keep the stable prefix?

Reply:
See my other comment (think it's fine to drop it here)

I _do_ wonder though, if we should have a link to the "archived" version of the docs for 17.06 (in case there's differences in the installation procedure; 

- https://docs.docker.com/v18.03/ee/supported-platforms/
- https://docs.docker.com/v17.06/engine/installation/ (but perhaps we need another page to link to)

Reply:
ok talked to @bermudezmt this looks resolved.